library/ecdh.c

```diff
@@ -144,9 +136,6 @@ int mbedtls_ecdh_setup( mbedtls_ecdh_context *ctx, mbedtls_ecp_group_id gid )
             ctx->point_format = MBEDTLS_ECP_PF_UNCOMPRESSED;
             ctx->var = MBEDTLS_ECDH_VARIANT_MBEDTLS_2_0;
             ctx->gid = gid;
-            mbedtls_ecp_group_init( &ctx->ctx.mbed_ecdh.grp );
-            mbedtls_ecp_point_init( &ctx->ctx.mbed_ecdh.Q   );
-            mbedtls_ecp_point_init( &ctx->ctx.mbed_ecdh.Qp  );
```

Comment:
This is for the new configuration and we might want to keep this in. Needs thinking about.

Reply:
I think it is simpler to not call these because
- this makes the initialisation more similar and less error prone
- this is how we did it in the past
- IIRC PSA mandates that `memset()` -ing to 0 is a valid initialisation of contexts so this will be a valid way of initialisation in the future too